src/ImageSharp.Drawing/Shapes/Polygon.cs

```diff
@@ -127,5 +135,21 @@ public Vector2[] AsSimpleLinearPath()
         {
             return this.innerPath.Points;
         }
+
+        /// <summary>
+        /// Finds the intersections.
```

Comment:
Can we expand this description slightly please. It's all still rocket science to me 🚀